Sloth.Sql/dbo/Tables/WebHookRequestResendJobRecords.sql

```diff
@@ -0,0 +1,11 @@
+CREATE TABLE [dbo].[WebHookRequestResendJobRecords] (
+    [Id]            NVARCHAR (450) NOT NULL,
+    [Name]          NVARCHAR (MAX) NULL,
+    [RanOn]         DATETIME2 (7)  NOT NULL,
+    [Status]        NVARCHAR (MAX) NULL,
```

Comment:
@sprucely Should status be smaller? varchar(20) or something like that?

Reply:
Copied if from Cybersource job. But yeah, I'll change it...